app/src/main/java/com/example/oddhours/ui/addjob/AddJobFragment.kt

```diff
@@ -103,11 +118,12 @@ class AddJobFragment : Fragment() {
                         "Job with this Name & Location exists already",
                         Toast.LENGTH_LONG
                     ).show()
-//                    clearFields()
                 }
             } else if (companyName == "") {
+                hideKeyboard()
```

Comment:
we should remove the `hidekeyboard()` functions from when job name or location is empty string. we're still throwing the error so i think it'd get pretty tiring to have to bring up the keyboard everytime that scenario happens